app/views/event_attendings/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Attend Event</h1>
+<%= form_for [@event, @event.event_attendings.build] do |f| %>
+<!--<%= f.collection_check_boxes :event_attendee_id, User.all, :id, :username %>-->
```

Comment:
- It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history